packages/autocomplete-plugin-algolia-insights/src/createAlgoliaInsightsPlugin.ts

```diff
@@ -304,3 +306,15 @@ function loadInsights(environment: typeof window) {
     console.error(errorMessage);
   }
 }
+
+/**
+ * While `search-insights` supports both string and number user tokens,
+ * the Search API only accepts strings. This function normalizes the user token.
+ */
+function normalizeUserToken(userToken?: string | number): string | undefined {
+  if (!userToken) {
+    return undefined;
+  }
```

Comment:
We're checking whether `userToken` is defined in l. 181 so we shouldn't have to check it again, right?

Reply:
You're right, thanks! I've updated the code.